sklearn/neighbors/quad_tree.pyx

```diff
@@ -19,19 +20,14 @@ import numpy as np
 cimport numpy as np
 np.import_array()
 
-cdef extern from "math.h":
-    float fabsf(float x) nogil
-
 cdef extern from "numpy/arrayobject.h":
     object PyArray_NewFromDescr(PyTypeObject* subtype, np.dtype descr,
                                 int nd, np.npy_intp* dims,
                                 np.npy_intp* strides,
                                 void* data, int flags, object obj)
 
 
-# XXX using (size_t)(-1) is ugly, but SIZE_MAX is not available in C89
-# (i.e., older MSVC).
-cdef SIZE_t DEFAULT = <SIZE_t>(-1)
+cdef SIZE_t DEFAULT = SIZE_MAX
```

Comment:
I think the referenced `SIZE_MAX` is the one from [`stdint.h`](https://en.wikibooks.org/wiki/C_Programming/stdint.h) but I would appreciate is someone could double-check.

Reply:
I can confirm it is defined there: https://en.cppreference.com/w/c/types/limits

Can we use `SIZE_MAX` directly in this file instead of defining a `DEFAULT`?